core/state_transition.go

```diff
@@ -296,7 +296,7 @@ func (st *StateTransition) TransitionDb() (ret []byte, usedGas uint64, failed bo
 	// Quorum - Privacy Enhancements
 	// perform privacy enhancements checks
 	if pmh.mustVerify() {
-		var exitEarly = false
+		exitEarly := false
```

Comment:
Not important but here the best would be

```go
var exitEarly bool
```

Declares variable that default to false which is the zero value of the bool type

Reply:
done